applications/nrf5340_audio/src/audio/audio_system.c

```diff
@@ -89,7 +94,14 @@ static void audio_headset_configure(void)
 	}
 #endif /* (CONFIG_STREAM_BIDIRECTIONAL) */
 
-	sw_codec_cfg.decoder.num_ch = SW_CODEC_MONO;
+	sw_codec_cfg.decoder.num_ch = SW_CODEC_ONE_CHANNEL;
+	sw_codec_cfg.decoder.channel_mode = SW_CODEC_MONO;
+
+	if (IS_ENABLED(CONFIG_SD_CARD_PLAYBACK)) {
+		/* Need an extra decoder channel to decode data from SD card */
+		sw_codec_cfg.decoder.num_ch++;
```

Comment:
Can we have the case where `num_ch` is 2, and we increment it to 3 here? The `sw_codec_num_ch` enum doesn't have a definition for 3, so we need to safeguard what happens in that case.